docs/components/CircleConfirmation.md

```diff
@@ -6,7 +6,27 @@ title: CircleConfirmation
 ## Example
 
 ```javascript
-<CircleConfirmation />
+
+onConfirmationCanceled = (event) => {
+    print("onConfirmationCanceled");
```

Comment:
`print` is a lumin only function, use `console.log` instead

Reply:
Ok, let me change and commit one more time.